packages/website/blog/2025-05-26-project-service.mdx

```diff
@@ -0,0 +1,226 @@
+
+The new Project Service API does support project references out-of-the-box.
+This is a huge win for monorepos, as it means you can lint all of your projects with type information without needing to create a separate `tsconfig.eslint.json` file for each project.
+
+## Performance
+
+Supporting project references allows the new Project Service API to be significantly faster than the traditional `parserOptions.project` API in many monorepo cases.
+We've observed improvements for typed linting speed in real-world repositories [^1] [^2].
+For smaller projects, the performance of the new Project Service API is similar to the traditional `parserOptions.project` API.
+
+When we first started working with the new project service API, it outperformed equivalent `parserOptions.project` setups by ~10-15%.
+Since then, we have observed regressions in performance that have brought it down in some cases to be slightly slower.
+
+We believe the new Project Service API should be faster than the traditional API, and are treating the lack of significant improvement as a bug.
+See [⚡ Performance: parserOptions.projectService sometimes no longer outperforms parserOptions.project](https://github.com/typescript-eslint/typescript-eslint/issues/9571) for more information.
+
+## Next Steps
+
+The new Project Service API is available in typescript-eslint v8.0.0 and later.
+We've been using it in our monorepo for over a year and have been thrilled to see many community repositories adopt it as well.
+
+### Giving Feedback
+
+We'd love to hear from you on how this option works for you.
+Does it live up to what we've promised, and/or does it have bugs we haven't fixed yet?
+Please do send us GitHub issues for any bugs you encounter or suggestions for how to improve the API.
+
+The [typescript-eslint Discord](https://discord.gg/FSxKq8Tdyg) is a great place to ask questions and engage with us more casually.
+For support in onboarding, feel free to ask in its `#help` channel.
+We'd be happy to help you try out `parserOptions.projectService` and learn more about how you use typescript-eslint.
+
+### Long Term Vision
+
+The new Project Service API is a great step towards making typed linting easier and more straightforward to configure.
+Our priority for the next year will be to improve the new Project Service API so that it works in all places the traditional Program API does.
+We won't remove the traditional project program behavior unless and until the new Project Service API is able to fully replace it.
+
+As of typescript-eslint@8.33.0, we've also extracted most of the Project Service code into a standalone [`@typescript-eslint/project-service`](/packages/project-service) package.
+It has no dependencies on ESLint and is designed to be usable for any linter to enable TypeScript's Project Service API for typed linting.
+See [Packages > Project Service](/packages/project-service) for more details.
```

Comment:
Note that https://github.com/typescript-eslint/typescript-eslint/issues/10383#issuecomment-2872962288 proposes a pretty big shake-up in other packages. I don't think it impacts this though. Just an FYI :)